flash/tabular/data.py

```diff
@@ -55,5 +55,5 @@ def embedding_sizes(self) -> list:
         # https://developers.googleblog.com/2017/11/introducing-tensorflow-feature-columns.html
         # The following "formula" provides a general rule of thumb about the number of embedding dimensions:
         # embedding_dimensions =  number_of_categories**0.25
-        emb_dims = [max(int(n ** 0.25), 16) for n in self.cat_dims]
+        emb_dims = [max(int(n**0.25), 16) for n in self.cat_dims]
```

Comment:
black's version update comes with a bonus 😄 (context: probably recent versions of black don't allow spaces around arithmetic operators)

Reply:
Nice! I was wondering why black keeps changing the format in my local env, but I was too lazy to look into this 😅 Thanks for having a look!